### ISSUE TITLE
docs: broken link to example vs-code settings.json config for zarf yaml schema validation

### ISSUE THREAD

**Issue by JoeHCQ1** [defenseunicorns/uds-common#189]
This link is broken: https://github.com/defenseunicorns/uds-common/blob/main/docs/development-ide-configuration.md?plain=1#L19. Note that link shows up twice in the file.

**Comment by zachariahmiller** [comment]
good catch! The file that is linked to is not `.vscode/settings.json` relative to the file its `../.vscode/settings.json`. Labelled this as `good first issue`. Similarly, the way the link is named could probably be adapted to be more accessible for screen reader/etc. For example, `[example](.vscode/settings.json)` could better identify what the link is something like `[example vscode settings file]`

**Comment by codyshoffner** [comment]
Stale Issue - Closing
